webapp/src/components/sidebar_buttons/sidebar_buttons.jsx

```diff
@@ -13,11 +13,9 @@ export default class SidebarButtons extends React.PureComponent {
         theme: PropTypes.object.isRequired,
         isTeamSidebar: PropTypes.bool,
         showRHSPlugin: PropTypes.func.isRequired,
-        issues: PropTypes.arrayOf(PropTypes.object),
-        inIssues: PropTypes.arrayOf(PropTypes.object),
-        outIssues: PropTypes.arrayOf(PropTypes.object),
+        countIssues: PropTypes.object,
         actions: PropTypes.shape({
-            list: PropTypes.func.isRequired,
+            count: PropTypes.func.isRequired,
```

Comment:
It seems that this prop is never used. And the original `list` prop was also not used